### PR TITLE
Update to polkadot v1.0.0

### PR DESCRIPTION
**Pull request by Moliholy**

### CODE REVIEW

**Review by peterwht — APPROVED** [review]

File: node/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "watr-node"
-version = "1.0.000"
+version = "1.0.0"
```
Comment (Moliholy) [inline node/Cargo.toml:3]: @peterwht why did you change this back? It was changed in the previous commit.

Comment (peterwht) [reply node/Cargo.toml:3]: `cargo fmt` didn't like it! Something about a `leading zero`
